projects/solar/src/fault_handler.c

```diff
@@ -40,7 +42,13 @@ StatusCode fault_handler_raise_fault(EESolarFault fault, uint8_t fault_data) {
     }
   }
 
-  StatusCode can_status = CAN_TRANSMIT_SOLAR_FAULT(fault, fault_data);
+  LOG_DEBUG("%d \n", (int)s_mppt_count);
+  StatusCode can_status = STATUS_CODE_EMPTY;
```

Comment:
IMO `STATUS_CODE_EMPTY` isn't the best default status code for the off chance that the mppt count is invalid/unspecified; maybe use `STATUS_CODE_UNINITIALIZED` if you want to use a nonzero code as the default.